src/main/java/de/taimos/gpsd4java/backend/SocketThread.java

```diff
@@ -64,7 +64,6 @@ public SocketThread(final BufferedReader reader, final GPSdEndpoint endpoint, fi
 		this.endpoint = endpoint;
 		this.resultParser = resultParser;
 		
-		this.setDaemon(true);
```

Comment:
I am not sure about this. Normally I like libraries to not mess with my applications lifecycle. When I am done I don't want other libs to prevent me from stopping.